### PR TITLE
vscode: support entering debug mode with F5 key in VSCode.

### PR DESCRIPTION
**Pull request by davidhopper2003**
add "docker/scripts/dev_start_gdb_server.sh" to start a gdbserver process out of the docker.
add "scripts/start_gdb_server.sh" to start a gdbserver process in the docker.
modify "vscode/tasks.json" and "vscode/launch.json" to support entering debug mode with F5 key in VSCode.

### CODE REVIEW

File: docker/scripts/dev_start_gdb_server.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline docker/scripts/dev_start_gdb_server.sh]: 2017 --> 2018

File: scripts/start_gdb_server.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline scripts/start_gdb_server.sh]: 2017 --> 2018

Comment (davidhopper2003) [reply scripts/start_gdb_server.sh]: OK.

File: docker/scripts/dev_start_gdb_server.sh (hunk 1 above)
Comment (davidhopper2003) [reply docker/scripts/dev_start_gdb_server.sh]: OK.

**Comment by davidhopper2003** [comment]
@lianglia-apollo I have changed the relevant files.

**Pull request merged**